### PR TITLE
Update usaco-760.mdx 

### PR DESCRIPTION
**Pull request by sathu-beep**
Jesse Chloe code was very messy and hard to understand + had a huge header for some reason??
Added my code which has detailed comments that explain the code in a simple manner and added neat variable names

### CODE REVIEW

File: solutions/usaco-760.mdx [hunk 1]
```diff
@@ -36,110 +36,63 @@ for i in past_order:
...
-		cout << id[i] << endl;
-	}
+	for(int i = 0; i < n; i++) cout << cowID[i] << ln; 
+}
 
+int main() {
+	#ifndef ONLINE_JUDGE
+	freopen("shuffle.in", "r", stdin);
+	freopen("shuffle.out", "w", stdout);
+	#endif
```
Comment (SansPapyrus683) [inline solutions/usaco-760.mdx:91]: remove these hashtags

File: solutions/usaco-760.mdx [hunk 2]
```diff
@@ -36,110 +36,63 @@ for i in past_order:
...
+			temp[e] = cowID[j-1];
 		}
+		// Change all the values of cowID array to get ready for the next iteration
+		for(int j = 0; j < n; j++) cowID[j] = temp[j]; 
 	}
 
-	For(i, 0, N){
-		cout << id[i] << endl;
-	}
+	for(int i = 0; i < n; i++) cout << cowID[i] << ln;
```
Comment (SansPapyrus683) [inline solutions/usaco-760.mdx:84]: spaces for the control flow statements please istg

File: solutions/usaco-760.mdx [hunk 3]
```diff
@@ -36,110 +36,63 @@ for i in past_order:
...
-				}
-			}
-			correct[idx] = id[j];
-		}
-		for(int j=0; j<N; j++){
-			id[j] = correct[j];
+	return 0;
+}
+ 
+void solve(){
```
Comment (SansPapyrus683) [inline solutions/usaco-760.mdx:61]: wait just put this in main what even

File: solutions/usaco-760.mdx [hunk 4]
```diff
@@ -36,110 +36,63 @@ for i in past_order:
...
-	int N; cin >> N;
-
-	vi a(N);
-
-	For(i, 0, N){
-		cin >> a[i];
+#define ln '\n'
+
+// Instantiating swapOrder array as global for easy access for custom find function 
+int swapOrder[100];
```
Comment (SansPapyrus683) [inline solutions/usaco-760.mdx:48]: make constant MAX_N with val 100
then declare this array like `int swapOrder[MAX_N]`

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
better but there's still room for improvement

**Review by dutinmeow — APPROVED** [review]
idk how to review stuff

File: solutions/usaco-760.mdx (hunk 2 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx:84]: wdym there are spaces...

Comment (SansPapyrus683) [reply solutions/usaco-760.mdx:84]:
```suggestion
for (int i = 0; i < n; i++)
	cout << cowID[i] << ln;
```

Comment (SansPapyrus683) [reply solutions/usaco-760.mdx:84]: and like
```cpp
for ([asdf]) {
}
```

File: solutions/usaco-760.mdx (hunk 4 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx (hunk 3 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx (hunk 2 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx (hunk 1 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx [hunk 5]
```diff
@@ -36,110 +36,62 @@ for i in past_order:
...
-
-	For(i, 0, N){
-		cin >> a[i];
-	}
-
-	vi id(N);
-
-	For(i, 0, N){
-		cin >> id[i];
+#define ln '\n'
```
Comment (maggie-j-liu) [inline solutions/usaco-760.mdx:45]: i don't think this is an approved macro 👀

File: solutions/usaco-760.mdx [hunk 6]
```diff
@@ -36,110 +36,62 @@ for i in past_order:
...
+int MAX_N = 100;
```
Comment (maggie-j-liu) [inline solutions/usaco-760.mdx:46]: should be `const`? since you use it as an array size

File: solutions/usaco-760.mdx [hunk 7]
```diff
@@ -36,110 +36,62 @@ for i in past_order:
...
+	int n; cin >> n;
+	int cowID[n];
+	for (int i = 0; i < n; i++) 
+		cin >> swapOrder[i];
+	for (int i = 0; i < n; i++) 
+		cin >> cowID[i];
+
+	// For loop iterates 3 times and simulates each swap backwards during every iteration
+	for (int i = 0; i < 3; i++){
+		// Temporary array to assist with the swapping as not to change the values of the cowID array
```
Comment (maggie-j-liu) [inline solutions/usaco-760.mdx:74]: these two comments are too long, split them into two lines

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/usaco-760.mdx (hunk 7 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx (hunk 6 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: done

File: solutions/usaco-760.mdx (hunk 5 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: removed

**Comment by sathu-beep** [comment]
fixed everything

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-760.mdx [hunk 8]
```diff
@@ -36,110 +36,69 @@ for i in past_order:
...
-			correct[idx] = id[j];
-		}
-		for(int j=0; j<N; j++){
-			id[j] = correct[j];
+int main() {
+	freopen("shuffle.in", "r", stdin);
+	freopen("shuffle.out", "w", stdout);
+	int n;
+	cin >> n;
+	int cowID[n];
```
Comment (maggie-j-liu) [inline solutions/usaco-760.mdx:67]: don't use variable length arrays

Comment (SansPapyrus683) [reply solutions/usaco-760.mdx:67]: just
just use vectors
they are so much better

Comment (sathu-beep) [reply solutions/usaco-760.mdx:67]: no vectors are to much work

Comment (sathu-beep) [reply solutions/usaco-760.mdx]: fixed and made it not variable

Comment (SansPapyrus683) [reply solutions/usaco-760.mdx]: sathu
vectors
work literally the same way
just
```cpp
vector<int> arr(n);
```

Comment (maggie-j-liu) [reply solutions/usaco-760.mdx]: fix the temp array too

Comment (sathu-beep) [reply solutions/usaco-760.mdx]: i fixed the temp array

Comment (maggie-j-liu) [reply solutions/usaco-760.mdx]: did you push your commit? i still see this 
image

File: solutions/usaco-760.mdx [hunk 9]
```diff
@@ -36,110 +36,69 @@ for i in past_order:
...
-
-	vi id(N);
-
-	For(i, 0, N){
-		cin >> id[i];
+const int MAX_N = 100;
+
+// Instantiating swapOrder array as global for easy access for custom find function 
+int swapOrder[MAX_N];
+int cowID[MAX_N];
```
Comment (sathu-beep) [inline solutions/usaco-760.mdx:49]: on my end @maggie-j-liu

File: solutions/usaco-760.mdx [hunk 10]
```diff
@@ -36,110 +36,69 @@ for i in past_order:
...
+	/* 
+	 * For loop iterates 3 times and simulates 
+	 * each swap backwards during every iteration
+	 */
+	for (int i = 0; i < 3; i++) {
+		/* 
+		 * Temporary array to assist with the swapping 
+		 * as not to change the values of the cowID array.
+		 */
+		int temp[n];
```
Comment (maggie-j-liu) [inline solutions/usaco-760.mdx:82]: yeah i see the cowID is changed, but not this array

Comment (sathu-beep) [reply solutions/usaco-760.mdx]: oop sorry fixed now

File: solutions/usaco-760.mdx (hunk 8 above)
Comment (sathu-beep) [reply solutions/usaco-760.mdx]: check below

**Review by SansPapyrus683 — APPROVED** [review]
still room for improvement, but at least it's not benq levels of bad

**Review by ar492 — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Review by nwatx — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
better

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**